app/views/dashboard/_activity.njk

```diff
@@ -32,12 +36,10 @@
     ]
   }) }}
 
-  <noscript>
-    {{ govukButton({
-      classes: 'nojs-update-button',
-      text: "Update"
-    }) }}
-  </noscript>
+  {{ govukButton({
+    classes: 'dashboard-date-range__select',
```

Comment:
Very minor but inconsistent use of `'` and `"` isn't ideal here

Reply:
Thanks - I will update that now